### PR TITLE
Send absolute path to stderr and stdout to the ensemble evaluator

### PR DESCRIPTION
**Pull request by mortalisk**

### CODE REVIEW

File: python/job_runner/reporting/event.py [hunk 1]
```diff
@@ -87,7 +89,14 @@ def _init_handler(self, msg):
                     "datacontenttype": "application/json",
                 },
                 {
-                    "jobs": [job.job_data for job in msg.jobs],
+                    "jobs": {
+                        job.index: {
+                            **job.job_data,
+                            "stderr": os.path.abspath(job.job_data["stderr"]),
+                            "stdout": os.path.abspath(job.job_data["stdout"]),
```
Comment (jondequinor) [inline python/job_runner/reporting/event.py]: plz use pathlib (`resolve`)

File: python/job_runner/reporting/event.py [hunk 2]
```diff
@@ -87,7 +89,14 @@ def _init_handler(self, msg):
...
+                        }
+                        for job in msg.jobs
```
Comment (jondequinor) [inline python/job_runner/reporting/event.py]: I find these multi line inline online dict expressions hard to read. Is it just me?

File: python/job_runner/reporting/event.py (hunk 1 above)
Comment (jondequinor) [inline python/job_runner/reporting/event.py]: Are these values always set?

File: python/job_runner/reporting/event.py (hunk 2 above)
Comment (mortalisk) [reply python/job_runner/reporting/event.py]: Don't know, but let's try on another size and see how it fits.

File: python/job_runner/reporting/event.py (hunk 1 above)
Comment (mortalisk) [reply python/job_runner/reporting/event.py]: Ok

Comment (mortalisk) [reply python/job_runner/reporting/event.py]: I'll add a guard in case not

**Review by jondequinor — APPROVED** [review]
🔪🐷🔥🥓

**Pull request merged**